src/test/java/unicash/model/ModelManagerTest.java

```diff
@@ -138,7 +137,7 @@ public void equals() {
 
         // TODO: Replicate this for transaction list
```

Comment:
I think this comment can be removed!

Reply:
I think that was added by @woojiahao. Can this be removed, Jia Hao? If so, I will remove it before merging the PR.

Reply:
^ Jia Hao has confirmed that it can be removed. Merging this PR will remove that line of comment